docs/README.md

```diff
@@ -0,0 +1,111 @@
+# Developer Documentation
+
+* [Installing the Boilerplate](#installing-the-boilerplate)
+* [NPM Scripts](#npm-scripts)
+* [Developing Locally](#developing-locally)
+* [Building for Production](#building-for-production)
+
+## Installing the Boilerplate
```

Comment:
"Installing" is the wrong word for this - change to something like "Starting a New Project"